packages/core/src/lib/select/searchable-select.svelte

```diff
@@ -88,26 +96,57 @@ const FOCUS_ITEM = 'focus-item';
 
 type MenuState = typeof CLOSED | typeof FOCUS_SEARCH | typeof FOCUS_ITEM;
 
+const detailedOptions = writable(optionsToDetailedOptions(options));
+$: detailedOptions.set(optionsToDetailedOptions(options));
```

Comment:
It was necessary because I had `searchValue` declared with `let` (which needed this to have a correct initial value).

Changing this (and `searchValue`) to a store seems to work as expected.

Reply:
Changed in fd0802e